adicionar atualização de senha

JoaoPedro0803 opened the pull request:
## Descrição
Implementa a parte de back-end da [US01](https://github.com/fga-eps-mds/2021-2-PUMA-Doc/issues/16)

JoaoPedro0803 committed:
adicionar atualização de senha

Mexazonic commented on dbconfig/dbConfig.js:
```diff
@@ -9,12 +9,17 @@ const dbSchema = require('./dbSchema');
-    rejectUnauthorized: false,
-  },
-});
+const settings = {connectionString,};
```
a declaração da variável settings deve ser feita utilizando let ao invés de const

JoaoPedro0803 committed:
Add files via upload

Correção na pasta dbConfig, linha 12 (const) ---> (let)

viniman27 committed:
1a parte da recuperação de senha com o email

Foi criada uma pasta emailService com a lógica necessária para a implementação da recuperação de senha com o email.

GabrielRoger07 committed:
2a parte da recuperação de senha com o email

Alteração nos arquivos UserRouter e UserController.

Gustavo-Nogueira committed:
resolvendo conflitos

Gustavo-Nogueira reviewed (approved):
Aceito !
